scripts/game_structure/game_essentials.py

```diff
@@ -660,13 +660,19 @@ def add_faded_offspring_to_faded_cat(self, parent, offspring):
 def load_manager(res: tuple):
     # initialize pygame_gui manager, and load themes
     manager = pygame_gui.ui_manager.UIManager(res, 'resources/defaults.json', enable_live_theme_updates=False)
+    #notosans font path
     manager.add_font_paths(
         font_name='notosans',
         regular_path='resources/fonts/NotoSans-Medium.ttf',
         bold_path='resources/fonts/NotoSans-ExtraBold.ttf',
         italic_path='resources/fonts/NotoSans-MediumItalic.ttf',
         bold_italic_path='resources/fonts/NotoSans-ExtraBoldItalic.ttf'
     )
+    #clangen font path
+    manager.add_font_paths(
+        font_name='clangen',
+        regular_path='resources/fonts/clangen.ttf'
```

Comment:
I see now. You successfully added a new font named "clangen" into the UIManager, all the text boxes have been themed to use "nonosans", by name.  To apply the custom font, you would need to load a different theme file for the new font, or load the clangen font with the font_name "notosans".  You would most likely also need to have bold and italic versions of the font, to make sure everything still works.